### PR TITLE
[redesign] use vite vue ssg instead

### PR DESCRIPTION
**Pull request by akrindev**
this PR made to use nuxtjs with tailwindcss and redesign the entire creasi.co landing page

### CODE REVIEW

**Comment by feryardiant** [comment]
Mantap!

Thank you @akrindev, bisa tolong struktur file nya ngikutin repo [`creasi/internal`](https://github.com/creasico/internals)?
- File `.editorconfig` jangan diilangin, bisa kopas punya [`internal`](https://github.com/creasico/internals/blob/master/.editorconfig)
- Penamaan file components pake `kebab-case`

**Comment by feryardiant** [comment]
Ada 2 dependency missing, `vue` dan `ufo`.

Screen Shot 2022-06-21 at 03 52 02

**Comment by feryardiant** [comment]
Partners gak muncul pas `pnpm preview`

Screen Shot 2022-06-21 at 06 38 18

Steps to reproduce, run:
1. `pnpm build`
2. `pnpm preview`

**Comment by akrindev** [comment]
#7 iconify has been added to devDependencies

File: src/modules/pwa.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+// https://github.com/antfu/vite-plugin-pwa#automatic-reload-when-new-content-available
+export const install: UserModule = ({ isClient, router }) => {
+  if (!isClient)
+    return
+
+  router.isReady().then(async () => {
+    const { registerSW } = await import('virtual:pwa-register')
+    registerSW({ immediate: true })
+  })
+}
```
Comment (feryardiant) [inline src/modules/pwa.ts:12]: These line returns error when run `pnpm build` with `vite-ssg build`.

Output:
```
[vite-ssg] An internal error occurred.
[vite-ssg] Please report an issue, if none already exists: https://github.com/antfu/vite-ssg/issues
file:///Path/To/Repo/.vite-ssg-temp/main.mjs:8
import { Workbox } from "workbox-window";
         ^^^^^^^
SyntaxError: Named export 'Workbox' not found. The requested module 'workbox-window' is a CommonJS module, which may not support all module.exports as named exports.
CommonJS modules can always be imported via the default export, for example using:

import pkg from 'workbox-window';
const { Workbox } = pkg;

    at ModuleJob._instantiate (node:internal/modules/esm/module_job:124:21)
    at async ModuleJob.run (node:internal/modules/esm/module_job:181:5)
    at async Promise.all (index 0)
    at async ESMLoader.import (node:internal/modules/esm/loader:281:24)
    at async build (file:///Path/To/Repo/node_modules/.pnpm/vite-ssg@0.20.2_zq52l2i4w6rtiwnfttyvqsjbxa/node_modules/vite-ssg/dist/chunks/build.mjs:171:87)
    at async Object.handler (file:///Path/To/Repo/node_modules/.pnpm/vite-ssg@0.20.2_zq52l2i4w6rtiwnfttyvqsjbxa/node_modules/vite-ssg/dist/node/cli.mjs:20:3)
 ELIFECYCLE  Command failed with exit code 1.
```

**Comment by feryardiant** [comment]
noticed tailwindcss-intellisense seems not working after the `tailwind.config` file been renamed to `.ts`. Initial indication is due to the tailwindcss-intellisense doesn't natively support `.ts` file.

@akrindev thoughts?

**Pull request merged**